go/libkb/globals.go

```diff
@@ -294,10 +295,12 @@ func (g *GlobalContext) ConfigureCaches() error {
 	g.CardCache = NewUserCardCache(g.Env.GetUserCacheMaxAge())
 	g.Log.Debug("Created CardCache, max age: %s", g.Env.GetUserCacheMaxAge())
 
-	// We consider the local DB as a cache; it's caching our
+	// We consider the local DBs as caches; they're caching our
 	// fetches from the server after all (and also our cryptographic
 	// checking).
-	g.LocalDb = NewJSONLocalDb(NewLevelDb(g))
+	g.LocalDb = NewJSONLocalDb(NewLevelDb(g, g.Env.GetDbFilename()))
+	g.LocalChatDb = NewJSONLocalDb(NewLevelDb(g, g.Env.GetChatDbFilename()))
```

Comment:
Maybe instead pass in a function that gets the filename? That way in case we ever want to a support a world in which that could get changed at runtime it would work, and also be closer to how it worked before this patch.